src/components/PermissionsManagement/UserPermissionsPopUp.jsx

```diff
@@ -45,6 +46,9 @@ function UserPermissionsPopUp({
 
   useEffect(() => {
     setUserPermissions(actualUserProfile?.permissions?.frontPermissions);
+    setUserRemovedDefaultPermissions(
+      actualUserProfile?.permissions?.removedDefaultPermissions?.front || [],
```

Comment:
```suggestion
      actualUserProfile?.permissions?.removedDefaultPermissions || [],
```
See comment on backend.